bittensor/dendrite.py

```diff
@@ -279,9 +279,9 @@ def __init__(self, neuron: bittensor_pb2.Neuron, config, keypair):
             }
         ) 
         # Loop back if the neuron is local.
-        if neuron.address == config.axon.remote_ip:
+        if neuron.address == config.axon.external_ip:
             ip = "localhost:"
-            if config.axon.remote_ip == "host.docker.internal":
+            if config.axon.external_ip == "host.docker.internal":
                 ip = "host.docker.internal:"
             self.endpoint = ip + str(neuron.port)
```

Comment:
do we really care about docker stuff anymore?

Reply:
no we don't, perhaps jsut get rid of this now